update/ansible/playbook/tasks/roles/initialization/tasks/main.yml

```diff
@@ -71,21 +71,41 @@
 
 - name: Update (both)
   block:
-    # This will implicitly create /srv/clickhouse
-    - name: Create clickhouse data directory
-      file:
-        path: "/srv/clickhouse/flags"
-        state: directory
-        owner: root
-        group: pmm
-        recurse: true
-
-    - name: Create empty file to convert clickhouse databases from ordinary to atomic
-      file:
-        path: "/srv/clickhouse/flags/convert_ordinary_to_atomic"
-        state: touch
-        owner: root
-        group: pmm
+    - name: Check that Clickhouse database engine is ordinary
+      stat:
+        path: /srv/clickhouse/flags/convert_ordinary_to_atomic
+      register: is_database_ordinary
+
+    - name: Convert clickhouse engine from ordinary to atomic
+      block:
+        # This will implicitly create /srv/clickhouse
+        - name: Create clickhouse data directory
+          file:
+            path: "/srv/clickhouse/flags"
+            state: directory
+            owner: root
+            group: pmm
+            recurse: true
+
+        - name: Create empty file to convert clickhouse databases from ordinary to atomic
+          file:
+            path: "/srv/clickhouse/flags/convert_ordinary_to_atomic"
```

Comment:
Why would we create it again, if that file already exists? I'm looking at line 108.

Reply:
good point